README.rst

```diff
@@ -49,7 +49,7 @@ repository::
 
     $ git clone https://github.com/astropy/pytest-doctestplus
     $ cd pytest-doctestplus
-    $ python ./setup.py install
+    $ pip install .
```

Comment:
Should we rather say `pip install -e .` here? I feel it's rather likely that when someone wants the dev version they also plan to tinker with it.

Reply:
No strong feeling, the test says "to install the latest development version from the source" so I didn't put the `-e`, but I could put it. Or remove the paragraph, people installing a dev version should know what they do ;)

Reply:
fair enough :)
Let's merge it then.